site2/docs/deploy-bare-metal.md

```diff
@@ -44,28 +43,25 @@ To run Pulsar on bare metal, the following configuration is recommended:
 
 > **Note**
 >
-> Broker is only supported on 64-bit JVM.
-
-> If you do not have enough machines, or to try out Pulsar in cluster mode (and expand the cluster later),
+> * Broker is only supported on 64-bit JVM.
+> * If you do not have enough machines, or to try out Pulsar in cluster mode (and expand the cluster later),
 > you can deploy a full Pulsar configuration on one node, where Zookeeper, the bookie and broker are run on the same machine.
```

Comment:
```suggestion
> * If you do not have enough machines, or you want to test Pulsar in cluster mode (and expand the cluster later), you can deploy Pulsar completely on one node, where ZooKeeper, bookie and broker are run on the same machine.
```

'on one node' and 'on the same machine'. Is there repetition? Can you modify it?